cmd/crc/cmd/console.go

```diff
@@ -28,17 +28,17 @@ var (
 )
 
 func init() {
-	consoleCmd.Flags().BoolVar(&consolePrintURL, "url", false, "Prints the OpenShift Web Console URL to the console.")
-	consoleCmd.Flags().BoolVar(&consolePrintCredentials, "credentials", false, "Prints the credentials which can be used to connect to the OpenShift Web Console.")
+	consoleCmd.Flags().BoolVar(&consolePrintURL, "url", false, "Print the URL for the OpenShift Web Console")
+	consoleCmd.Flags().BoolVar(&consolePrintCredentials, "credentials", false, "Print the credentials for the OpenShift Web Console")
 	rootCmd.AddCommand(consoleCmd)
 }
 
 // consoleCmd represents the console command
 var consoleCmd = &cobra.Command{
 	Use:     "console",
 	Aliases: []string{"dashboard"},
-	Short:   "Opens or displays the OpenShift Web Console URL.",
-	Long:    `Opens the OpenShift Web Console URL in the default browser or displays it to the console.`,
+	Short:   "Access the OpenShift Web Console",
```

Comment:
'Access' is not what is happening here. Access is to enter a place.
I'd rather see this being `Open the OpenShift Web Console in the default browser`

Reply:
"Open the OpenShift Web Console in the default browser" would be great if that was the only thing the console command did. But it can also print out the console URL and credentials without launching a browser. With "access" I was hoping to capture all these functions, either with a browser getting launched outright, or providing the user the information to plug into a browser manually.

Reply:
> But it can also print out the console URL and credentials without launching a browser. 

If options are given ... access would still not convey that same meaning in a short description.